README.md

```diff
@@ -16,11 +16,19 @@ We use a Dart CLI application to easily create new projects.
 
 ### Generate a new project
 
-1. Install the CLI using this command.
+1. Fetch all dependencies.
+
+   Change directory to src/app in the terminal, and run these following commands:
+
+   `flutter pub get`
```

Comment:
This should be after creating a project, those steps aren't needed for the CLI when it will be fixed.